### DISCUSSION TITLE
How to use tkinter?

### DISCUSSION THREAD

**Discussion by Daftical56** [community/community#119787] [Codespaces]
### Select Topic Area

Bug

### Body

Hello people:

I've tried to use the tkinter library with Python on GitHub Codespaces, but I don't know how to configure an output port.

How could I configure an output port to access the window?

 > PS: I'm not a very advanced developer, so I need concise and clear steps _(sorry, I don't know certain things that some devs consider common sense._)

Attentive to new answers,
@**Daftical56**

**Comment by ciozru** [comment]
tkinter so hard library
